src/components/Lend/LoanSearch/LoanSearchFilter.vue

```diff
@@ -180,3 +212,9 @@ export default {
 	},
 };
 </script>
+
+<style lang="scss" scoped>
+	.filter-min-w {
```

Comment:
In future work we may move this up onto the template as a style attribute instead.